rust/benches/pack_index.rs

```diff
@@ -1,44 +1,42 @@
-#![feature(test)]
```

Comment:
Feel free to simply delete this file. It's not run in CI, and I only used it for a small amount of time.